### PR TITLE
Move shaderc::CompileOptions to test fixture.

### PR DESCRIPTION
**Pull request by dekimir**

### CODE REVIEW

File: libshaderc/src/shaderc_cpp_test.cc [hunk 1]
```diff
@@ -79,7 +80,10 @@ class CppInterface : public testing::Test {
   // message.
   std::string CompilationErrors(const std::string& shader,
                                 shaderc_shader_kind kind,
-                                const shaderc::CompileOptions& options) {
+                                // This could default to options_, but that can
+                                // be easily confused with a no-options-provided
+                                // case:
```
Comment (dneto0) [inline libshaderc/src/shaderc_cpp_test.cc]: Good comment.

File: libshaderc/src/shaderc_cpp_test.cc [hunk 2]
```diff
@@ -79,7 +80,10 @@ class CppInterface : public testing::Test {
   // message.
   std::string CompilationErrors(const std::string& shader,
```
Comment (Qining) [inline libshaderc/src/shaderc_cpp_test.cc]: Change it to CompilationWarnings() or I change it later?

Comment (Qining) [inline libshaderc/src/shaderc_cpp_test.cc]: Change the function name to CompilationWarnings() or I can change it and add another CompilationErrors() later?

Comment (dneto0) [inline libshaderc/src/shaderc_cpp_test.cc]: It's a separate issue not related to this PR.  Was discussed in that other PR.

Comment (Qining) [inline libshaderc/src/shaderc_cpp_test.cc]: OK, then +2.

**Pull request merged**